### PR TITLE
Ocrvs 3423

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
Updating padding of TableView heading and content wrapper causes this result on Status table

status-table

Other results

image

image

image

### CODE REVIEW

File: packages/client/src/views/Performance/FieldAgentList.tsx [hunk 1]
```diff
@@ -327,6 +329,13 @@ function FieldAgentListComponent(props: IProps) {
     ]
   }
 
+  const getFieldAgentTypeLabel = useCallback(
+    (type) => {
+      return userMessages[type] ? intl.formatMessage(userMessages[type]) : type
+    },
+    [intl]
+  )
+
```
Comment (rikukissa) [inline packages/client/src/views/Performance/FieldAgentList.tsx:338]: I think a plain function would suffice in this case as the function itself isn't passed down as a prop. useCallback is more for the case where you have a `const onClick = (event) =>` type of a handler that is created in the render function and you don't want the receiving component to get a new function reference on every render. Was it eslint that was suggesting this approach?

**Review by rikukissa — CHANGES_REQUESTED** [review]

File: packages/client/src/views/Performance/FieldAgentList.tsx (hunk 1 above)
Comment (tahmidrahman-dsi) [reply packages/client/src/views/Performance/FieldAgentList.tsx:338]: No, I just thought it would assign new function value every render otherwise

Comment (rikukissa) [reply packages/client/src/views/Performance/FieldAgentList.tsx:338]: Okay 👍 In this case it's fine as you're not passing the function reference to any component as a prop

Comment (tahmidrahman-dsi) [reply packages/client/src/views/Performance/FieldAgentList.tsx:338]: Understood @rikukissa, thanks very much. Doing the changes

**Review by rikukissa — APPROVED** [review]

**Pull request merged**